src/scanner/scanner.py

```diff
@@ -193,7 +194,7 @@ def _get_alerta_table(self) -> pd.DataFrame:
     def _filter_city(self, geocode):
         dfcity = self.data[self.data.municipio_geocodigo == geocode].copy()
         dfcity.sort_index(inplace=True)
-        dfcity["casos_cum"] = dfcity.casos.cumsum()
+        # dfcity["casos_cum"] = dfcity.casos.cumsum()
```

Comment:
any reason to keep this comment in the code? if it's not used anymore, it can be removed